### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by over-geek**
## Portfolio Project: Desktop Version

In this milestone, I:
- Replaced hamburger menu with nav links
- Added media queries to CSS to make the page responsive
- Changed the flex-direction of the card contents to row
- Modified background-position and background-size properties of the background image of the contact form
- Used percentages in padding to make the page responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -235,23 +246,27 @@ <h4 id="about-heading">Let's Connect</h4>
...
+                                        <p>HTML</p>
+                                    </div>
+                                </li>
+                                <li class="language-item">
+                                    <div>
+                                        <img src="asset/css-icon.png" alt="css icon">
+                                        <p>CSS</p>
+                                    </div>
+                                </li>
+                            </ul>
```
Comment (muneebulrehman) [inline index.html:269]: This part of your application is not matching the Figma template of your choosing. The icons are smaller in the Figma template placed in the top left corner. The box containing the image and text should have a `border-radius: 8px` which is missing in your case. Kindly fix these issues.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

Comment (muneebulrehman) [inline index.html:269]: The **About Myself and **Languages** section should be more or less on the same line as per the Figma template. However, in your case, the **Languages** heading has a bit more padding which is pushing it a bit more down from the top line. Kindly fix this issue.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @over-geek , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Consider removing this part from your README file. ⬇️
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — COMMENTED** [review]
# Hello @over-geek    😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 


### OPTIONALSUGGETION
- Kindly consider putting the below into consideration to make your readme professional
[OPTIONAL]op



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**